docs-src/php/how-to-handle-workflow-logic-requirements-in-php.md

```diff
@@ -22,7 +22,7 @@ Always do the following in the Workflow implementation code:
 - Do not use any blocking SPL provided by PHP (i.e. `fopen`, `PDO`, etc) in **Workflow code**.
 - Use `yield Workflow::getVersion()` when making any changes to the Workflow code. Without this, any deployment of updated Workflow code
   might break already open Workflows.
-- Don’t access configuration APIs directly from a Workflow because changes in the configuration might affect a Workflow execution path.
+- Don’t access configuration APIs directly from a Workflow because changes in the configuration might affect a Worklfow Execution path.
```

Comment:
```suggestion
- Don’t access configuration APIs directly from a Workflow because changes in the configuration might affect a Workflow Execution path.
```